scheduler/src/cook/mesos/dru.clj

```diff
@@ -31,18 +28,11 @@
   [db running-task-ents pending-job-ents]
   (timers/time!
     init-user->dru-divisors-duration
-    (let [all-running-users (->> running-task-ents
-                               (map util/task-ent->user)
-                               (into #{}))
-        all-pending-users (->> pending-job-ents
-                               (map :job/user)
-                               (into #{}))
-        all-users (clojure.set/union all-running-users all-pending-users)
-        user->dru-divisors (->> all-users
-                                (map (fn [user]
-                                       [user (share/get-share db user)]))
-                                (into {}))]
-    user->dru-divisors)))
+    (let [all-running-users (map util/task-ent->user running-task-ents)
+          all-pending-users (map :job/user pending-job-ents)
+          all-users-set (-> #{} (into all-running-users) (into all-pending-users))
```

Comment:
As long as we're optimizing, would it be faster to `(set (concat all-running-users all-pending-users))`?